test/fixtures/strings-de-msgctxt.po

```diff
@@ -0,0 +1,113 @@
+msgid ""
+msgstr ""
+"Plural-Forms: nplurals=2; plural=(n != 1);\n"
+"X-Crowdin-Project: paperless-app\n"
+"X-Crowdin-Project-ID: 405180\n"
+"X-Crowdin-Language: de\n"
+"X-Crowdin-File: strings.pot\n"
+"X-Crowdin-File-ID: 10\n"
+"Project-Id-Version: paperless-app\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Language-Team: German\n"
+"Language: de_DE\n"
+"PO-Revision-Date: 2021-01-04 16:55\n"
+
+msgid "Error while connecting to server"
+msgstr "Fehler beim Verbinden mit dem Server"
+
+#: msgctxt "some context for this string"
+msgid "Retry"
+msgstr "Wiederholen"
+
+#: msgctxt "another context for this string"
+msgid "Edit Server Details"
+msgstr "Serverdetails bearbeiten"
+
+#: msgctxt "some context for this string"
+msgid "Error while uploading document"
+msgid_plural "Error while uploading document"
+msgstr[0] "Fehler beim Hochladen des Dokument"
+msgstr[1] "Fehler beim Hochladen des Dokuments"
+
+msgid "My Documents"
+msgstr "Meine Dokumente"
+
+msgid "Settings"
+msgstr "Einstellungen"
+
+#: msgctxt "another context for this string"
+msgid "Logout"
+msgstr "Abmelden"
+
+#: msgctxt "another context for this string"
```

Comment:
Sorry, but I'm confused now. Shouldn't this test for `msgctx` without the comment?

Reply:
No not in this case, it's what I said earlier, in  the .po file it's correct with the comment. Only in the .pot file it's without. The .pot file is the output of the getstrings command.

Reply:
After carefully reading the .PO file spec, I still don't think that's quite right. A search on GitHub leads to a lot of results without the comment: https://github.com/search?q=msgctxt+extension%3Apo&type=code. Are you sure about that?

Reply:
Interesting, I thought Poedit outputted the files this way, but after testing this again the output is as expected without comments. Interestingly, the parser still handles both cases completely fine. I will update the tests nonetheless, since the correct way seems to be writing it without the comment per specification and examples you provided.